kit/database.go

```diff
@@ -78,6 +78,34 @@ func database(args []string) error {
 		}
 
 		fmt.Println("✅ Database dropped successfully")
+
+	case "reset":
+		err := db.Drop(url)
+		if err != nil {
```

Comment:
I wonder if we should ignore the error on Drop. Would it err if the DB is not there?

Reply:
Oh! I just noticed the sqlite and postgres implementations don't return an error if there's no database :D